packages/dashmate/test/unit/templates/dynamicComposeBuildArgs.spec.js

```
@@ -0,0 +1,65 @@
+import { expect } from 'chai';
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import dot from 'dot';
+
+const __filename = fileURLToPath(import.meta.url);
```

ESLint

[error] Unexpected dangling '_' in '__filename'.

(no-underscore-dangle)